modules/server/src/network/types/hits.ts

```diff
@@ -0,0 +1 @@
+export type Hits = { total: number };
```

Comment:
there is not a TS definition for these so adding as I go

Reply:
Will this file have more types added? Wondering the need for its own file if it's only a one line definition

Reply:
Very likely and definitely ideally. It related to a core Arranger schema type which has many more properties than this.
As long as we get to it , it should be the seperate entity types as types which compose into this parent `Hits` type